tests/testthat/test-xamples.R

```diff
@@ -289,17 +275,15 @@ testthat::test_that("Test Dataset 8 builds correctly", {
       file.path(examples_dir, "Test_2023_8/metadata.yml"),
       file.path(examples_dir, "Test_2023_8/data.csv"),
       "Test Dataset 8", definitions, unit_conversions, schema, resource_metadata, taxon_list
-    ),
-    info = "Building Test Dataset 8")
+    ))
 
   # Expected output
   tables <- c("traits", "locations", "contexts", "methods", "excluded_data",
               "taxonomic_updates", "taxa", "contributors")
   expect_no_error(
     expected_output <-
       purrr::map(
-        tables, ~read_csv(sprintf("examples/Test_2023_8/output/%s.csv", .x), col_types = cols(.default = "c"))),
-    info = "Reading in expected output tables"
```

Comment:
Why delete? The info isn't helpful?

Reply:
It's using `expect_no_error` from {testthat} which does not accept an info argument.

Reply:
Ok, but you also had a new function defined called `expect_no_error`, which would overwrite the one from {test that}. Now it's renamed to `test_ expect_no_error`, so perhaps isn't being used?

Reply:
Oh there shouldn't have been a new function defined called `expect_no_error`; if there was- that's my bad! 

I can't seem to find it though. I did a find and replace on `test_` to `test_expect_` before so theoretically I shouldn't have touched `expect_no_error`.